department-of-veterans-affairs/caseflow
# Allow judge to judge assignment of ama cases

Resolves #7612 

### Description
Allow judges to reassign ama cases to another judge

### Acceptance Criteria
- [x] On the unassigned cases page, allow a judge to reassign a task to another judge
- [x] On the case details cases page for an usassigned task, allow a judge to reassign the task to another judge

### Testing Plan
1. Checkout this branch and reseed the database with `bundle exec rake db:setup`
2. Switch to the user "BVAEBECKER"
3. Open your queue and navigate to assign cases
4. Select a task, select "Other" from the "Assign to:" dropdown, and select "Judge with hearings and cases" from the "Enter the name of an attorney or judge:" dropdown.
5. Hit "Assign 1 case" and notice no errors (This case should then also appear in BVAAABSHIRE's assign cases view)
